drink-cli/src/app_state/mod.rs

```diff
@@ -69,7 +69,7 @@ impl Default for UiState {
 }
 
 pub struct AppState {
-    pub session: Session<MinimalRuntime>,
+    pub session: Session<MinimalSandbox>,
```

Comment:
I think it's kinda weird that `Session` is now parameterised with the `Sandbox` - it encloses the sandbox but the parameter type should be something that was the common denominator for both - session and sandbox. 

Cannot point my finger to anything specific, so this won't be a blocker, but I just have a gut feeling that previous typing was more "appropriate".

Reply:
> it encloses the sandbox but the parameter type should be something that was the common denominator for both

The sandbox itself enclose the runtime so the common denominator still stands